Lumberjack.xcodeproj/project.pbxproj

```diff
@@ -2392,14 +2391,14 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
-				MACOSX_DEPLOYMENT_TARGET = 10.8;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```

Comment:
Why not 10.12?

Reply:
Cause we have ~~Installer which is 10.11 😅~~ Lib which is 10.10 :hidethepain:.

Reply:
Oh, sure, makes sense